app/models/server/models/LivechatRooms.js

```diff
@@ -81,6 +82,22 @@ export class LivechatRooms extends Base {
 		return this.update(query, update);
 	}
 
+	updateBatchDataByToken(token, fields) {
+		const query = { 'v.token': token, 'open': true };
+		const room = this.findOne(query, { fields: { livechatData: 1 } });
+		const updates = {};
+
+		fields.map((f) => {
+			if (!f.overwrite && room.livechatData && typeof room.livechatData[f.key] !== 'undefined') {
+				return null;
+			}
+			updates[`livechatData.${f.key}`] = f.value;
+			return null;
+		});
```

Comment:
```suggestion
		fields.forEach((f) => {
			if (!f.overwrite && room.livechatData && typeof room.livechatData[f.key] !== 'undefined') {
				return;
			}
			updates[`livechatData.${f.key}`] = f.value;
		});
```

Reply:
Since we're not converting/mapping an array here, we can use `forEach` loop instead.